core/src/test/java/tech/pegasys/web3signer/core/multikey/metadata/AwsKeySigningMetadataDeserializerTest.java

```diff
@@ -37,20 +33,10 @@
   private final String AWS_VALID_CONFIG_SPECIFIED_AUTH_MODE_PATH =
       "src/test/resources/aws/aws_valid_config_specified.yaml";
 
-  private ObjectMapper objectMapper;
-
-  @BeforeEach
-  public void before() {
-    objectMapper = new ObjectMapper(new YAMLFactory());
-    SimpleModule module = new SimpleModule();
-    module.addDeserializer(AwsKeySigningMetadata.class, new AwsKeySigningMetadataDeserializer());
-    objectMapper.registerModule(module);
-  }
-
   @Test
```

Comment:
Do you need to register your deserializer still?

Reply:
I'm not doing it anymore. These lines were deleted :)